include/openthread/commissioner.h

```diff
@@ -110,6 +110,18 @@ typedef struct otCommissioningDataset
     bool mIsJoinerUdpPortSet : 1; ///< TRUE if Joiner UDP Port is set, FALSE otherwise.
 } otCommissioningDataset;
 
+/**
+ * This structure represents a Joiner Info.
+ *
+ */
+typedef struct otJoinerInfo
+{
+    otExtAddress mEui64;                 /// Joiner eui64
+    char         mPsk[OT_PSKC_MAX_SIZE]; /// Joiner pskc
+
+    bool mAny : 1; /// TRUE if eui64 isn't set, FALSE otherwise.
```

Comment:
Suggestion: How about adding the expiration time as well? We can return how many more second till joiner will be timed out and removed?